docs/en/logs/index.asciidoc

```diff
@@ -0,0 +1,17 @@
+:doctype: book
+:metrics: infrastructure
+:metrics-app: Infrastructure app
+:logs: logs
+:logs-app: Logs app
```

Comment:
Should App be capitalized? If so, this is probably a separate issue as it applies to more than just the Logs App (and this PR). https://wiki.elastic.co/display/MAR/Term+usage+guide

Reply:
Thanks for doing the review at short notice! 

No, I don't think "app" should be capitalised. In general, we avoid capitals unless things really are proper names, or there is anther very good reason for using them. As far as I'm aware, using "app" agrees with the marketing guidelines.